pypesto/optimize/optimizer.py

```diff
@@ -790,3 +795,78 @@ def nlopt_objective(x, grad):
 
     def is_least_squares(self):
         return False
+
+
+class FidesOptimizer(Optimizer):
+    """
+    Global/Local optimization using the trust region optimizer fides.
```

Comment:
add a link to the github page (or rtd, when exists)?

Reply:
added